pkg/netutil/latency/edge/derp.go

```diff
@@ -30,7 +30,7 @@ func measureDERP(ctx context.Context, targetDerpServrs *tailcfg.DERPMap, opts ..
 		return nil, err
 	}
 
-	log.Logger.Debugw("measuring latencies to public tailscale DERP nodes")
+	log.Logger.Infow("measuring latencies to public tailscale DERP nodes")
```

Comment:
shall we print it just in up or join sub-command?

Reply:
done